xdebug/README.md

```diff
@@ -58,10 +58,28 @@ If you're not sure which `php.ini` file to edit, you can check that by executing
 php --ini
 ```
 Make sure you restart PHP-FPM server after making these changes. If you followed this development setup,
-then reload commands should look something like this:
+then reload commands should look something like this on Mac:
 ```
 sudo port reload php74-fpm
 ```
+or like this on Ubuntu:
+```
+sudo systemctl reload php7.4-fpm
+```
+
+### If using Ubuntu
+
+The congifuration to enable the debugger may not work on Ubuntu when added to `php.ini`.
+If debugging doesn't work and the XDebug section of `phpinfo` says that the Step Debugger is disabled,
+try adding the configuration to the XDebug-specific configuration file. The file will have a name similar
+to `20-xdebug.ini`. This file should appear in the printout of the command
+```
+php --ini
+```
+If there is no such configuration file, make sure that XDebug is enabled by running
+```
+sudo phpenmod xdebug
```

Comment:
We can also use `sudo phpenmod -v ALL -s ALL xdebug` to enable the extension on all SAPI's (`-s ALL `), usually CLI and FPM, and all versions (`-v ALL`).

You can also document the disabling too: `phpdismod` instead of `phpenmod`

Reply:
> You can also document the disabling too: `phpdismod` instead of `phpenmod`

This is already documented at the end of the .md file, I just copied the command here because I thought it might be relevant: If XDebug is disabled, the `20-xdebug.ini` symlink isn't created and doesn't appear in the printout of `php -ini`.

> We can also use `sudo phpenmod -v ALL -s ALL xdebug` to enable the extension on all SAPI's (`-s ALL `), usually CLI and FPM, and all versions (`-v ALL`).

This then seems a little out of scope of this pull request, perhaps it could be added in the dedicated section on enabling and disabling XDebug at the end of the guide?

Reply:
Do however you think is best, I'm okay with it either way :)

In any case, my intention was to elminate the eventual discrepancies where for one PHP version, xdebug would work, but not for the other one and so on.

Reply:
I have updated the commands to include the `-v ALL -s ALL` flags in 71a9c5e4f3ac11c41306e733788e94da7f977ae5, can you check if they look okay?

Reply:
Looks good :+1: